build-parent/pom.xml

```diff
@@ -68,7 +68,8 @@
         <microprofile-opentracing-api.version>1.3.3</microprofile-opentracing-api.version>
         <microprofile-context-propagation.version>1.0.1</microprofile-context-propagation.version>
         <microprofile-jwt-api.version>1.1.1</microprofile-jwt-api.version>
-
+        <microprofile-graphql.version>1.0.2</microprofile-graphql.version>
```

Comment:
Maybe add the `-api` suffix? Looks like they are all consistent except context-propagation.